.cspell/cspell.config.yml

```
@@ -10,134 +11,97 @@ dictionaryDefinitions:
     path: ./names.txt
     scope: workspace
     addWords: true
-  # Not all french word are present in the `fr-fr` dictionary.
-  - name: french-words
-    path: ./french-words.txt
-    addWords: true
-  # `filetypes` is missing some file types
   - name: custom-ext
+    description: File types missing in the `filetypes` dictionary
     path: ./custom-ext.txt
     addWords: true
-  - name: jinja
-    path: ./jinja.txt
-    addWords: true
+# Dictionaries to always be used
 dictionaries:
+  # custom dictionaries
+  - custom-words
+  - custom-ext
+  - names
+  # cspell dictionaries
+  - bash
+  - cpp
+  - cpp-refined
+  - docker
   - en_us
+  - en-common-misspellings
   - en-gb
-  - fr-fr
-  - french-words
+  - filetypes
+  - fonts
+  - html
+  - latex
   - lorem-ipsum
+  - makefile
   - misc
-  - softwareTerms
   - networking-terms
+  - npm
+  - npm
+  - powershell
   - python
   - python-common
   - rust
-  - cpp
-  - cpp-refined
-  - filetypes
-  - bash
   - shell
-  - custom-words
-  - filetypes
-  - custom-ext
-  - names
-  - typescript
+  - shellscript
+  - softwareTerms
   - sql
+  - typescript
   - win32
-  - jinja
-words:
-  - libparsec
-  - schemafile
-  - intlify
-  - htmlcov
-  - Swatinem
-ignoreWords:
-  - ABCDEFGHJKLMNPQRSTUVWXYZ
-  - Aaaaaand
-  - AIAA
-  - ApiV
-  - Assss
-  - devel
-  - Crefreshiconhandler
-  - Cwindowsiconhandler
-  - DWXJ
-  - EFGH
-  - HRSW
-  - JKLM
-  - KBWM
-  - NNWQ
-  - NPQR
-  - PQRSTUVWXYZHIJKLMNO
-  - QPGW
-  - QPIM
-  - QWIPQs
-  - QYJW
-  - SPFYHIZLEL
-  - SPIHB
-  - SYMED
-  - VFKA
-  - VQLP
-  - WPFF
-  - XBAO
-  - XPCMTSEN
-  - YVPZ
-  - ZGIQ
-  - ZOXKGGIMTY
-  - abcefg
-  - ackup
-  - csdef
-  - dbmdl
-  - dbproj
-  - dksjdskdjs
-  - ebug
-  - elease
-  - eleases
-  - esult
-  - iobj
-  - ipch
-  - ipdb
-  - jmconfig
-  - mnpcod
-  - ntvs
-  - nunit
-  - nupkg
-  - nvuser
-  - paket
-  - pidb
-  - psess
-  - rsuser
-  - snupkg
-  - splitn
-  - tqdm
-  - Ssjfq
 cache:
   cacheFormat: universal
   cacheStrategy: metadata
-  useCache: true
+  useCache: false
```

Oops, this was disabled for testing, it should be re-enabled
```suggestion
  useCache: true
```